osu.Game/Screens/Play/Player.cs

```diff
@@ -87,7 +87,14 @@ public class Player : ScreenWithBeatmapBackground, IProvideCursor
         private FailOverlay failOverlay;
 
         private DrawableStoryboard storyboard;
-        private Container storyboardContainer;
+        protected UserDimContainer StoryboardContainer;
```

Comment:
```suggestion
        protected UserDimContainer StoryboardContainer { get; private set; }
```

Along with all other protected members in this class.